registration.php

```diff
@@ -1,407 +1,874 @@
-                              <br>
                            </div>
-                           <br><br>
+                           <br>
+                           <br>
                            <div class="form-group">
-                              <label class="control-label col-xs-4" for="office_fax">Fax:</label>
+                              <label class="control-label col-xs-4" for="office_fax">Fax:
+                          </label>
                               <div class="col-xs-8">
-                                 <input type="text" class="form-control" id="office_fax" name="office_fax" placeholder="44-444-4444">
+                                 <input type="text" class="form-control" id="office_fax" name="office_fax" value="<?php echo (isset($office_fax) ? $office_fax : ''); ?>" maxlength="10" >
+                                 <span class="error"> 
+                                 <?php echo "<br>";
+                                  echo isset($_SESSION['error']['office_fax_err'])?  $_SESSION['error']['office_fax_err'] : '';
+                                 ?> 
+                                 </span>
                               </div>
                            </div>
                         </div>
                         <!--  end of row -->
                      </div>
                      <hr class="hr" />
-                     <h2><u>OTHER DETAILS:</u></h2>
+                     <h2>
+                        <u>OTHER DETAILS:
+                        </u>
+                     </h2>
                      <br />
                      <div class="form-group">
-                        <label class="control-label col-lg-3" for="comm">Preferred Communication:</label>
+                        <label class="control-label col-lg-3" for="comm">Preferred Communication:
+                        </label>
                         <div class="col-lg-9">
-                           <label class="checkbox-inline"><input type="checkbox" name="communication[]" value="mobile" />Mobile</label>
-                           <label class="checkbox-inline"><input type="checkbox" name="communication[]" value="sms" />SMS</label>
-                           <label class="checkbox-inline"><input type="checkbox" name="communication[]" value="email" />Email</label>
-                           <label class="checkbox-inline"><input type="checkbox" name="communication[]" value="other" />Other</label>
+                           <label class="checkbox-inline">
+                           <input type="checkbox" name="communication[]" value="mobile" 
+                              <?php if(isset($comm_1)&&$comm_1=="mobile") echo "checked"; ?> />Mobile
+                           </label>
+                           <label class="checkbox-inline">
+                           <input type="checkbox" name="communication[]" value="sms"  
+                              <?php if(isset($comm_2)&&$comm_2=="sms") echo "checked"; 
```

Comment:
Remove this dead/unused code and check everywhere if there are any such codes and remove .